### DISCUSSION TITLE
[Support]: Detector Temperature Reporting

### DISCUSSION THREAD

**Discussion by greyman12** [blakeblackshear/frigate#11657] [General Support]
### Describe the problem you are having

I have both a USB and a M.2 Coral installed, from my understanding only the M.2 will report the temp. Within the UI I am seeing the temp reporting for the USB and not the PCI coral. Nothing major just something I noticed.

### Version

dev-8c32580

### Frigate config file

```yaml
n/a
```

### Relevant log output

```shell
n/a
```

### FFprobe output from your camera

```shell
n/a
```

### Frigate stats

```json
n/a
```

### Operating system

Proxmox

### Install method

Docker CLI

### Coral version

USB

### Network connection

Wired

### Camera make and model

n/a

### Any other information that may be helpful

![image](https://github.com/blakeblackshear/frigate/assets/65574532/fa204c4c-136e-49cf-b03c-35993df1872c)

**Comment by NickM-27** [comment]
there is no way to perfectly correlate which detector is showing the temperatures. In this case it is the pcie detector but we don't know which name that relates to when building these stats. This is an edge case that I don't think will be handled
